### DISCUSSION TITLE
Package.json being shown as unknown file extension

### DISCUSSION THREAD

**Discussion by Tooba-QM** [thegetty/quire#773] [Get Help] [answered]
Hi,
After installing quire, I wanted to check the version. But this error is being shown (Please see image below). I have followed the installation guide step-by-step. And have tried to reinstall quire too but it did not work. 

(Error shown in powershell)
![quire version error ](https://github.com/thegetty/quire/assets/136605222/0b8e85a5-6b8d-4fcb-9d69-0fde4792ec9b)

Could anyone please help me with this?

I would be thankful to you.

**Comment by Erin-Cecele** [comment, marked as answer]
Thank you, @Tooba-QM. We'll review this error message and get back to you shortly.  In the meantime, can you confirm you are using the LTS version of Node.js?

**Comment by Tooba-QM** [comment]
Hi,

The problem was solved once I installed the latest version of Node.js

Thank you.

________________________________
From: Erin Cecele Dunigan ***@***.***>
Sent: Tuesday, July 11, 2023 3:10:09 AM
To: thegetty/quire ***@***.***>
Cc: Tooba Aziz ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL]Re: [thegetty/quire] Package.json being shown as unknown file extension (Discussion #773)


Thank you, @Tooba-QM<https://github.com/Tooba-QM>. We'll review this error message and get back to you shortly. In the meantime, can you confirm you are using the LTS version of Node.js?

**Reply by Erin-Cecele** [reply]
That's wonderful new @Tooba-QM. Let us know if you need anything else.
